fixed installed tab was not sticky

MSchmoecker opened the pull request:
closes #527. Everything seems to work fine, guess it was just legacy styling?

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ebkr/r2modmanPlus?pullRequest=531) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ebkr/r2modmanPlus?pullRequest=531) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ebkr/r2modmanPlus?pullRequest=531) it.</sub>

ebkr commented:
Hi, thanks for the PR!
This however was to align the installed sections no mods installed message properly so that rather than being dead center, it was centered within the block to the right of the navigation

MSchmoecker commented:
Ah I see. I moved the `class="relative-position full-height--minus-em"` as a new div inside and added `v-if="localModList.length === 0"` there as well

ebkr commented:
Thanks for the quick fix! I'll get around to reviewing it ASAP which should be tomorrow at the latest, although it all looks fine from what the file changes.
